lib/ExpensiMark.js

```diff
@@ -61,16 +61,8 @@ export default class ExpensiMark {
                 // Use the url escaped version of a backtick (`) symbol. Mobile platforms do not support lookbehinds,
                 // so capture the first and third group and place them in the replacement.
                 // but we should not replace backtick symbols if they include <pre> tags between them.
-                regex: /(\B|_|)&#x60;(?:(?!(?:(?!&#x60;).)*?<pre>))(.*?\S.*?)&#x60;(\B|_|)(?!&#x60;|[^<]*<\/pre>)/g,
-                replacement: (match, g1, g2, g3) => {
-                    const regex = /^[&#x60;]+$/i;
-
-                    // if content of the inline code block is only backtick symbols, we should not replace them with <code> tag
-                    if (regex.test(g2)) {
-                        return match;
-                    }
-                    return `${g1}<code>${g2}</code>${g3}`;
-                },
+                regex: /(\B|_|)&#x60;(.*?(?![&#x60;])\S.*?)&#x60;(\B|_|)(?!&#x60;|[^<]*<\/pre>)/gm,
```

Comment:
I think this change might've caused the following issue:
- https://github.com/Expensify/App/issues/46917

Which was fixed in PR https://github.com/Expensify/expensify-common/pull/785, more details in the [proposal](https://github.com/Expensify/App/issues/46917#issuecomment-2272779877).